Constructive Code Review/outline.txt

```diff
@@ -0,0 +1,205 @@
+- Splash screen
+    - PR screenshot: “Your code is bad and you are bad. Have a bad day.”
+- Importance
+    - Who's done code review before?
+        - Who's done it sync?
+            - We don't really do that, aside from maybe some pair
+              programming.
+        - Async?
+    - Pretty much all code at Mozilla gets reviewed before it gets
+      merged.
```

Comment:
While DXR (and Input and like half of what I've worked on) sometimes gets YOLO-ey, I think those are edge-of-the-edge-case exceptions to the rule and it's probably better to sweep them under the carpet and pretend they didn't exist unless we want to get into why those projects are "valid exceptions" and how to assess risk in code changes and things like that.